### DISCUSSION TITLE
How to get the XrmToolBox.AutoUpdater.exe to work?

### DISCUSSION THREAD

**Discussion by ASDF-PS** [MscrmTools/XrmToolBox#1289] [General]
Hi,

I am not sure how to get the autoupdater.exe to work correctly? steps to reproduce.
Extract the application zip toC:\ProgramData\XrmToolbox
When an update is avaliable run the autoupdate feature via the GUI, it prompts to close the app, then fails. (running as local admin also fails)

If I try via command line without the app open it still fails.
If I run it from the root of C it backs up the folders \ files in that location. If I run it from the XRMToolbox folder I get the same error as the GUI

C:\ProgramData\XrmToolbox>"C:\ProgramData\XrmToolBox\X
![Error](https://github.com/MscrmTools/XrmToolBox/assets/53652632/8f0191e7-a120-413f-ace8-34698f561c26)
rmToolBox.AutoUpdater.exe" https://github.com/MscrmTools/XrmToolBox/releases/download/v1.2023.9.66/XrmToolbox.zip C:\ProgramData\XrmToolbox\XrmToolBox.exe

Is there any chance of a winget repo being provided?

**Comment by BetimBeja** [comment]
On the shared screenshot you have a very long path 😮 Try unpacking/running in a shorter path... maybe C:\XrmToolBox 

I don't know if @MscrmTools has any plans of providing a Winget repo.

**Reply by MscrmTools** [reply]
I don't have yet this in my backlog... and I know nothing about winget so I would need to study it first.

As for the autoupdater exe. As its name should explain it, it's for auto update. You shouldn't have to run it by yourself.

As @BetimBeja says, this path is very long and I don't know why your run XrmToolBox from a backup folder of a backup folder of a...

Please consider downloading again XrmToolBox and install it in a specific folder.

**Comment by ASDF-PS** [comment]
Dropped it in the root of C and it works fine.

Thanks for your time both. Much appreciated.
